File: python/solutions/part1/longest_word_finder.py
sentence_string = "This is a long sentence with words of multiple lengths for inputting into your function"
longest_word = ''

words = sentence_string.split()
for word in words:
	if len(word) > len(longest_word):
		longest_word = word

print 'Longest word is {} at a length of {}'.format(longest_word, len(longest_word))

File: java/solutions/part1/LongestWordFinder.java
public class LongestWordFinder {
  public static void main(String[] args) {
    String mySentence = "Hello, I am a sentence with many different word lengths.";
    String[] words = mySentence.split(" ");

    String longestSoFar = "";
    for (int i = 0; i < words.length; i++) {
      if (words[i].length() > longestSoFar.length()) {
        longestSoFar = words[i];
      }
    }

    System.out.println(longestSoFar);
  }
}

